resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterManager.scala

```diff
@@ -46,8 +46,6 @@ private[spark] class KubernetesClusterManager extends ExternalClusterManager wit
       sc: SparkContext,
       masterURL: String,
       scheduler: TaskScheduler): SchedulerBackend = {
-    val executorSecretNamesToMountPaths = KubernetesUtils.parsePrefixedKeyValuePairs(
-      sc.conf, KUBERNETES_EXECUTOR_SECRETS_PREFIX)
```

Comment:
It does not create any side effects so it can be removed.

Reply:
Do we not have secrets -> mountpaths support right now? @mccheah @liyinan926

Reply:
@foxish you are right we do have, but this statement has no effect. It was used in the past, the following statement was removed:
```
-    val mountSecretBootstrap = if (executorSecretNamesToMountPaths.nonEmpty) {	
-      Some(new MountSecretsBootstrap(executorSecretNamesToMountPaths))	
-    } else {	
-      None	
-    }
```
https://github.com/apache/spark/commit/a83ae0d9bc1b8f4909b9338370efe4020079bea7#diff-7d9979c0153744eafa24348ecbfa1671

Reply:
@mccheah @liyinan926 any more to this?

Reply:
@mccheah I think the logic has changed otherwise the tests I have in this PR(https://github.com/apache/spark/pull/21652)  would have failed when removed that part and re-run them.